client/src/app.scss

```
@@ -355,7 +355,7 @@ sup.new {
 
 .sticky-header-container {
   position: sticky;
-  top: 0;
+  top: -1px;
```

FWIW `-0.1px` seems to be sufficient.

```suggestion
  top: -0.1px;
```